databases/week2/Working with tasks.sql

```diff
@@ -0,0 +1,49 @@
+
+	-- Write the following sql queries:--
+	-- 1- Add a task with the these attributes: title, description, created, updated, dueDate, statusID, userID --
+		INSERT INTO `task` (`title`, `description`,`created`, `updated`, `due_Date`,`status_ID`, `user_ID`) 
+		VALUES ('complete_homework_week 2', 'Its about db', '2019-11-18 19:31:00','2019-11-18 20:31:00','2019-12-18 20:31:00', '3','8');
+
+	-- 2- Change the title of a task with these attributes: taskID, newTitle --
+		update task set title = 'Do some shopping for dark winter' where id = 1;
+
+	-- 3-Change the task due date with these attributes: taskID, newDueDate -- 
+		update task SET due_date = '2019-12-31 20:58:03' WHERE task.id = 1;
+
+	-- 4-Change the task status with these attributes: taskID, newStatus -- 
+		UPDATE task SET status_id = 2 WHERE id = 1;
+
+	-- 5-Mark a task as complete with this attribute: taskID -- 
+		UPDATE task SET status_id = 3 WHERE id = 5;
+
+	-- 6- Delete task with this attribute: taskID -- 
+		DELETE FROM task WHERE id = 36;
+
+
+	-- SCHOOL DATABASE!! -- 
+	-- 1- Create a new database containing the following tables: -- 
+	-- Class: with the columns: id, name, begins (date), ends (date) -- 
+	-- Student: with the columns: id, name, email, phone, class_id (foreign key)-- 
+
+		create database school_db;
+		use school_db; 
+		create table `Class` (`id` int(10) unsigned not null auto_increment, `name` varchar(255) not null,`begins` DATETIME NULL DEFAULT NULL,`ends` DATETIME NULL DEFAULT NULL);
```

Comment:
You should indent the columns to avoid long lines and help readability. It would also be better to name the table lowercase `class`. Then the naming is consistent everywhere.

Furthermore i'm getting an error from MySQL:

> Error in query (1075): Incorrect table definition; there can be only one auto column and it must be defined as a key

This is because you have forgotten to specify the id column as primary key.